### PR TITLE
Tweaked native tokens explanation with more details

### PR DESCRIPTION
**Pull request by olgahryniuk**
@JaredCorduan 
This PR adds the content which we have on Cardano Docs (such as a getting started tutorial, native tokens exercises, exchange multi-asset management scenarios, and listing advice) and also includes some styling tweaks to existing sections. Can you please review and merge if happy? I also made changes to the rst index file so that new sections are included in the user-facing documentation.

### CODE REVIEW

File: doc/explanations/exchange-scenarios.rst [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+2. Run the following curl command to check and verify the wallet
+   information:
+
+**Byron**
+
+::
+
+   curl http://localhost:8090/v2/byron-wallets/ 
+
+**Shelley**
```
Comment (JaredCorduan) [inline doc/explanations/exchange-scenarios.rst]:
```suggestion
**Shelley and beyond**
```

File: doc/explanations/token-bundles.rst [hunk 2]
```diff
@@ -94,19 +103,18 @@ This is an example of a token bundle, let’s call it ``TB_Example`` : ::
 			(VIPTickets, 50)}
 	}
 
-We will use this example to explain the terminology below.
 
 How and where are token bundles stored?
 ********************************************
 
-Token bundles can be found:
+Token bundles can be found:
```
Comment (JaredCorduan) [inline doc/explanations/token-bundles.rst]:
```suggestion
Token bundles can be found:
```

File: doc/explanations/exchange-scenarios.rst [hunk 3]
```diff
@@ -0,0 +1,244 @@
+=========================================
+Exchange multi-asset management scenarios
```
Comment (JaredCorduan) [inline doc/explanations/exchange-scenarios.rst]: This looks like some really excellent documentation, but it is uses a lot of components downstream of the ledger:
* cardano-graphql
* cardano-rosetta
* cardano-db-sync
* cardano-wallet

The folks working in this repository (cardano-ledger) will not know when things fall out of date, and this documentation will very likely quickly become invalid if it lives here.

File: doc/explanations/exercises.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
+=======================
+Native tokens exercises
```
Comment (JaredCorduan) [inline doc/explanations/exercises.rst]: This documentation should probably live in the cardano-node repository.

File: doc/explanations/faq.rst [hunk 5]
```diff
@@ -1,218 +1,337 @@
...
+transactions. These transactions must also carry the minting policies
+for the tokens they are minting, so that they can be checked during
+validation.
+
+The result of processing a minting transaction is that the ledger will
+now additionally contain the assets included in the mint field (minting
+field) of the transaction. If the quantity of a particular asset in the
+mint field is negative, the result is that after processing the
+transaction, the total quantity of that specific asset on the ledger
+will be reduced by the amount reflected in the mint field.
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst]:
```suggestion
A. Every ledger era starting with Mary contains a field in the transaction
body for minting multi-assets, named the mint field.
In order to use the minting field, transaction must be authorized by the
minting policy. Positive values create assets, and negative ones destroy them.
```

File: doc/explanations/faq.rst [hunk 6]
```diff
@@ -1,218 +1,337 @@
...
+
+This set of rules is defined by the user who wishes to create the new
+asset. For example, a user might wish to only allow themselves to ever
+mint this particular kind of token. In this case, they would stipulate
+this in the policy. The node checks adherence to minting policies when a
+transaction is processed by running the code or checking the relevant
+signatures. Transaction data must satisfy all the minting policies of
+all assets the transaction is attempting to mint.
+
+**Q. What is a token builder and what is its functionality?**
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst]: The token builder is not a part of the ledger, so this explanation will need to live elsewhere, unfortunately.

File: doc/explanations/faq.rst [hunk 7]
```diff
@@ -1,218 +1,337 @@
...
-################################################
+===========================================
 
-What is 'multisig' and how is it related to minting policies?
-********************************************************************************
+**Q. What is ‘multisig’ and how is it related to minting policies?**
 
-The multisig scripting language (which existed prior to the introduction of MA functionality on Cardano) specifies some minimal set of signatures required to allow a transaction to perform a certain action, usually to spend a UTxO entry.
+A. The multisig scripting language (which existed prior to the
+introduction of MA functionality on Cardano) specifies some minimal set
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst]: the multi-sig DSL was actually introduced in the Shelley ledger era.

File: doc/explanations/faq.rst [hunk 8]
```diff
@@ -1,218 +1,337 @@
...
 
-What is a one-time minting policy?
-********************************************************************************
+**Q. What is a one-time minting policy?**
 
-In a one-time minting policy, the complete set of tokens scoped under it is minted by one specific transaction. This means that no more tokens will ever be minted under that policy. This type of policy does require smart contracts and cannot be expressed using multisig.
+A. In a one-time minting policy, the complete set of tokens scoped under
+it is minted by one specific transaction. This means that no more tokens
+will ever be minted under that policy. This type of policy does require
+smart contracts and cannot be expressed using multisig.
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst:135]: The multi-sig DSL was expanded in the Allega ledger era to include timelocks, which do provide the ability to guarantee that a given policy can never again be used after a gives slot.

File: doc/explanations/faq.rst [hunk 9]
```diff
@@ -1,218 +1,337 @@
...
+real-world diamonds. If there are no other assets a token is fungible
+with -such as a token representing a house- the token is deemed to be
+unique (non-fungible).
 
-What is a token bundle?
-********************************************************************************
-
-A mixed collection of tokens scoped under one or more minting policies. Any tokens can be bundled together.
+**Q. What is a token bundle?** A. A mixed collection of tokens scoped
+under one or more minting policies. Any tokens can be bundled together.
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst:159]: do you want to mention asset IDs?

File: doc/explanations/faq.rst [hunk 10]
```diff
@@ -1,218 +1,337 @@
...
 For more detail, see the token bundle section.
 
-
 Transacting with native tokens
-###################################
+==============================
 
-How do native tokens appear in a user's wallet?
-********************************************************************************
+**Q. How do native tokens appear in a user’s wallet?**
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst]: there is no notion of a "wallet" in the ledger, so this description should not live here.

File: doc/explanations/faq.rst [hunk 11]
```diff
@@ -1,218 +1,337 @@
...
-************************************************************************************
+**Q. What types of assets can I use to cover costs associated with
+native tokens?**
 
-Currently, only ada can be used to make fee payments or to satisfy the min-ada-value constraint.
+A. Currently, only ada can be used to make fee payments or deposits.
 
-How does coin selection work for custom native tokens?
-********************************************************************************
+**Q. How does coin selection work for custom native tokens?**
```
Comment (JaredCorduan) [inline doc/explanations/faq.rst]: coin selection really has nothing to do with the ledger, but really a wallet implementation detail.

File: doc/explanations/get-started.rst [hunk 12]
```diff
@@ -0,0 +1,719 @@
+==================================
+Getting started with native tokens
```
Comment (JaredCorduan) [inline doc/explanations/get-started.rst]: This documentation should probably live in the cardano-node repository.

File: doc/explanations/exercises.rst (hunk 4 above)
Comment (olgahryniuk) [reply doc/explanations/exercises.rst]: Noted, thanks! I'll delete it from here and talk to the node team to host this in 'references'

File: doc/explanations/exchange-scenarios.rst (hunk 3 above)
Comment (olgahryniuk) [reply doc/explanations/exchange-scenarios.rst]: Noted, thanks! I'll delete this explainer from the ledger repo and see where we can host it.

File: doc/explanations/faq.rst (hunk 6 above)
Comment (olgahryniuk) [reply doc/explanations/faq.rst]: Deleted, thanks!

File: doc/explanations/faq.rst (hunk 7 above)
Comment (olgahryniuk) [reply doc/explanations/faq.rst]: Tweaked to avoid inaccuracy

File: doc/explanations/faq.rst (hunk 9 above)
Comment (olgahryniuk) [reply doc/explanations/faq.rst:159]: I think we have a nice explainer in the dedicated section (assets, tokens, and token bundles). No need to write too much here.

File: doc/explanations/faq.rst (hunk 10 above)
Comment (olgahryniuk) [reply doc/explanations/faq.rst]: deleted

File: doc/explanations/faq.rst (hunk 11 above)
Comment (olgahryniuk) [reply doc/explanations/faq.rst]: deleted

File: doc/explanations/get-started.rst (hunk 12 above)
Comment (olgahryniuk) [reply doc/explanations/get-started.rst]: deleted

**Comment by olgahryniuk** [comment]
@JaredCorduan I think I have now addressed all your suggestions. Thanks so much for going through this! Please take a look if everything is okay now and if the index.rst is properly edited.

**Review by JaredCorduan — APPROVED** [review]
looks great, thank you @olgahryniuk !

**Pull request merged**